lib/ansible/module_utils/facts/system/distribution.py

```diff
@@ -64,8 +64,8 @@ class DistributionFiles:
         {'path': '/etc/system-release', 'name': 'Amazon'},
         {'path': '/etc/alpine-release', 'name': 'Alpine'},
         {'path': '/etc/arch-release', 'name': 'Archlinux', 'allowempty': True},
-        {'path': '/etc/os-release', 'name': 'SuSE'},
-        {'path': '/etc/SuSE-release', 'name': 'SuSE'},
+        {'path': '/etc/os-release', 'name': 'SUSE'},
+        {'path': '/etc/SuSE-release', 'name': 'SUSE'},
```

Comment:
If they changed their name, is the path `/etc/SuSE-release` still valid? Did they ever change it to `/etc/SUSE-release`? If so, should we add that as an additional path here?

Reply:
No, that path name was never changed, it has been replaced with /etc/os-release only.